modules/express/src/clientRoutes.ts

```diff
@@ -1004,6 +1009,23 @@ export async function handleV2EnableTokens(req: express.Request) {
   }
 }
 
+/**
+ * Handle Update Wallet
+ * @param req
+ */
+async function handleWalletUpdate(req: express.Request): Promise<unknown> {
+  // If it's a lightning coin, use the lightning-specific handler
+  if (isLightningCoinName(req.params.coin)) {
+    return handleUpdateLightningWalletCoinSpecific(req);
+  }
```

Comment:
If the client wants to update other wallet settings, `handleUpdateLightningWalletCoinSpecific` can not handle it.

If we want to go in this direction, `handleUpdateLightningWalletCoinSpecific` should be modified to `handleUpdateLightningWallet` and have logic to pass Record<string, unknown> along with `coinSpecific` to backend.